modules/rosa-hcp-aws-privatelink-create-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-aws-privatelink-create-cluster_{context}"]
+= Creating an AWS PrivateLink cluster
+
+You can create a private cluster on {hcp-title} using the {product-title} (ROSA) CLI, `rosa`.
+
+[NOTE]
+====
+AWS PrivateLink is supported on existing VPCs only.
+====
+
+.Prerequisites
+
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your installation host.
+* You have created a VPC with at least one private subnet.
+
+.Procedure
+
+Creating a cluster with {hcp} can take around 10 minutes.
+
+. You can create a {hcp-title} cluster with multiple availability zones (Multi-AZ). Your machine's classless inter-domain routing (CIDR) must match your virtual private cloud's CIDR. See https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc[Requirements for using your own VPC] and link:https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-validation_installing-aws-vpc[VPC Validation] for more information.
```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps), we should use the imperative mood in the first sentence of each step. Additionally, I don't see a clear action for a user to take in this step. 

I'd suggest either rewriting this step as an instruction to ensure the CIDR matches for this condition, or I'd move this paragraph to the prereqs, as something to ensure for people who intend to create a cluster with multiple availability zones (or whatever the condition is)

Reply:
I agree